### ISSUE TITLE
nom install后有异常

### ISSUE THREAD

**Issue by p2ng** [hexojs/hexo#3215]
## Environment Info

Node version(`node -v`):
v10.7.0

Your site `_config.yml` (Optional):

Your theme `_config.yml` (Optional): 

Hexo and Plugin version(`npm ls --depth 0`): 
├── hexo@3.7.1
├── hexo-deployer-git@0.3.1
├── hexo-generator-archive@0.1.5
├── hexo-generator-category@0.1.3
├── hexo-generator-index@0.2.1
├── hexo-generator-tag@0.2.0
├── hexo-renderer-ejs@0.2.0
├── hexo-renderer-marked@0.2.11
├── hexo-renderer-stylus@0.3.3
├── hexo-server@0.2.2
└── hexo-wordcount@3.0.2

## For BUG
- nom install
```
> fsevents@1.2.4 install /Users/liangzp/Document/source/p2ng.github.io/node_modules/fsevents
> node install

[fsevents] Success: "/Users/liangzp/Document/source/p2ng.github.io/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile

> nunjucks@3.1.3 postinstall /Users/liangzp/Document/source/p2ng.github.io/node_modules/nunjucks
> node postinstall-build.js src

added 509 packages from 523 contributors and audited 2312 packages in 14.635s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
```

- nom audit
```

                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ uglify-js                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.6.0                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ hexo-deployer-git                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ hexo-deployer-git > swig > uglify-js                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/48                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 2312 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

## For question
查看依赖`hexo-deployer-git`有`swig`但是`node_modules`里没有`swig`文件夹、只有`swig-extras`、`swig-templates`
查看`node_modules`里有`uglify-js`而且版本是v2.6.0；

**Comment by bojue** [comment]
发自我的iPhone------------------ 原始邮件 ------------------发件人: 梁展鹏  <notifications@github.com>发送时间: 2018年7月27日 16:17收件人: hexojs/hexo <hexo@noreply.github.com>抄送: Subscribed <subscribed@noreply.github.com>主题: 回复：[hexojs/hexo] nom install后有异常 (#3215)
Environment Info
Node version(node -v):
v10.7.0
Your site _config.yml (Optional):
Your theme _config.yml (Optional):
Hexo and Plugin version(npm ls --depth 0):
├── hexo@3.7.1
├── hexo-deployer-git@0.3.1
├── hexo-generator-archive@0.1.5
├── hexo-generator-category@0.1.3
├── hexo-generator-index@0.2.1
├── hexo-generator-tag@0.2.0
├── hexo-renderer-ejs@0.2.0
├── hexo-renderer-marked@0.2.11
├── hexo-renderer-stylus@0.3.3
├── hexo-server@0.2.2
└── hexo-wordcount@3.0.2
For BUG

nom install

> fsevents@1.2.4 install /Users/liangzp/Document/source/p2ng.github.io/node_modules/fsevents
> node install

[fsevents] Success: "/Users/liangzp/Document/source/p2ng.github.io/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile

> nunjucks@3.1.3 postinstall /Users/liangzp/Document/source/p2ng.github.io/node_modules/nunjucks
> node postinstall-build.js src

added 509 packages from 523 contributors and audited 2312 packages in 14.635s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details


nom audit


                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ uglify-js                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.6.0                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ hexo-deployer-git                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ hexo-deployer-git > swig > uglify-js                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/48                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 2312 scanned packages
  1 vulnerability requires manual review. See the full report for details.

For question

For feature request


—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/hexojs/hexo","title":"hexojs/hexo","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/hexojs/hexo"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"nom install后有异常 (#3215)"}],"action":{"name":"View Issue","url":"https://github.com/hexojs/hexo/issues/3215"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/hexojs/hexo/issues/3215",
"url": "https://github.com/hexojs/hexo/issues/3215",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "nom install后有异常 (#3215)",
"sections": [
{
"text": "",
"activityTitle": "**梁展鹏**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@p2ng",
"facts": [
{
"name": "Repository: ",
"value": "hexojs/hexo"
},
{
"name": "Issue #: ",
"value": 3215
}
]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"hexojs/hexo\",\n\"issueId\": 3215,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close issue",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueClose\",\n\"repositoryFullName\": \"hexojs/hexo\",\n\"issueId\": 3215\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/hexojs/hexo/issues/3215"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 361522474\n}"
}
],
"themeColor": "26292E"
}
]

**Comment by YoshinoriN** [comment]
@p2ng 
Thanks. I created PR to hexo-deployer-git repository.
